src/Expensify.tsx

```diff
@@ -121,7 +121,7 @@ function Expensify() {
     const shouldHideSplash =
         shouldInit &&
         (NativeModules.HybridAppModule
-            ? splashScreenState === CONST.BOOT_SPLASH_STATE.READY_TO_BE_HIDDEN && (isAuthenticated || !!hybridApp?.useNewDotSignInPage)
+            ? splashScreenState === CONST.BOOT_SPLASH_STATE.READY_TO_BE_HIDDEN && (isAuthenticated || useNewDotSignInPage)
```

Comment:
```suggestion
            ? splashScreenState === CONST.BOOT_SPLASH_STATE.READY_TO_BE_HIDDEN && (isAuthenticated || !!useNewDotSignInPage)
```

I think we still need `!!` here to get rid of eslint error